server/src/routes/api/users.js

```diff
@@ -160,27 +160,53 @@ router.post('/:id/coursehistory', async (req, res) => {
  */ 
 
 router.get('/:id/coursehistory', async (req, res) => {
-  let userId = req.params.id;
+  let userId = 1;
 
   if (await user.getCourses(userId) <= 0){
     console.log('no course history found for user');
     res.status(200).send('no course history found for user');
   } else {
     const courses = await user.getCourses(userId); 
-    console.log(courses);
-    res.status(200).send("fetching all user courses: " + courses);
+    console.log(courses[0]);
+    res.status(200).send({message: "fetching all user course history", course: courses});
   }
 });
 
 /**
- * @route POST api/logout
- * @desc end the users session
+ * @route GET api/users/userinfo
+ * @desc Retreive all user information
  * @access Private
- */
+ */ 
+
+router.get('/:id/userinfo', async (req, res) => {
```

Comment:
standard way to access an item via the api would be `api/items/idofitem` so in this case it shoud be api/users/:id